src/css/style.css

```
@@ -1,63 +1,108 @@
 :root {
-  --black: #0D1117;
-  --gray: #8B949E;
+  --gray: #C4C4C4;
+  --black: #000000;
+  --white: #FFFFFF;
+  --gray-op-1: #C9D1D9;
+  --button-dark-blue: #001E66;
+  --button-light-blue: #76A4FF;
+  --input-border-blue: #3878F9;
   --gray-op-2: rgb(139, 148, 158, 0.2); 
-  --white: #C9D1D9;
 }
-
 body {
-  background-color: var(--black); 
-  color: var(--white);
   font-size: 1.1rem;
+  color: var(--white);
+  background-color: var(--white);
 }
-
 #app {
+  height: 97vh;
   display: flex;
-  height: 100vh;
-  justify-content: center;
   align-items: center;
+  justify-content: center;
 }
-
 .input, .fieldset {
-  background-color: var(--black);
-  border: 1px solid var(--gray);
-  border-radius: 5px;
-  color: var(--gray);
+  width: 220px;
+  height: 38px;
+  margin-top: 3px;
   padding: 5px 12px;
   font-size: 1.1rem;
+  color: var(--black);
+  border-radius: 10px;
+  margin-bottom: -5px;
+  font-family: Montserrat;
+  border: 1px solid var(--gray);
+  background-color: var(--white);
+}
+.input:focus {
+  outline-width: 0px;
+  border-color: var(--input-border-blue);
+}
+.div {
+  text-align: center;
 }
-
 .btn {
-  background-color: #238636;
+  width: 75px;
   border: none;
-  color: #FFF;
+  height: 27px;
+  padding: auto;
   cursor: pointer;
-  padding: 5px 12px;
-  border-radius: 5px;
+  margin: 5px 8px;
+  color:var(--white);
+  margin-bottom: 0px;
```

consider mergin `margin` rules using shorthand, for example:

```suggestion
  margin: 5px 8px 0;
  color:var(--white);
```